core-api/src/main/java/com/hevelian/identity/core/api/TenantController.java

```diff
@@ -66,11 +70,20 @@ public void deactivateTenant(@Valid @RequestBody TenantDomainDTO tenantDomainDTO
         tenantService.addTenant(tenant.toEntity(), tenant.getTenantAdmin().toEntity()).getDomain());
   }
 
+  @RequestMapping(path = "/setTenantLogoBytes", method = RequestMethod.POST)
```

Comment:
There is no need of this method on a Controler level.

Reply:
Fixed